src/ray/raylet/placement_group_resource_manager.h

```diff
@@ -53,8 +53,9 @@ class PlacementGroupResourceManager {
   /// Lock the required resources from local available resources. Note that this is phase
   /// one of 2PC, it will not convert placement group resource(like CPU -> CPU_group_i).
   ///
-  /// \param bundle_spec: Specification of bundle whose resources will be prepared.
-  virtual bool PrepareBundle(const BundleSpecification &bundle_spec) = 0;
+  /// \param bundle_specs: Specification of bundle whose resources will be prepared.
```

Comment:
Can you update what the return value means? Especially something like "if the all bundles cannot be prepared, it returns false"